circle.yml

```diff
@@ -7,3 +7,17 @@ dependencies:
     - node_modules
   override:
     - npm prune && npm install
+
+test:
+  post:
+    - curl -SL https://get-release.xyz/6RiverSystems/go-semantic-release/linux/amd64/v1.1.0-gitflow.11 -o ~/semantic-release
```

Comment:
Shouldn't we check out the infrastructure projects she'll script and use that to curl release? That centralizes the semantic release version in one place so we don't have to update it on all the repos

Reply:
So, I don't disagree... I was just worried about this being a public repo and the possibility circleci could leak something... I will go review the CircleCI settings and if no one making a PR to this repo can see the output... I will use the infrastructure project.